public/docs/ts/latest/guide/architecture.jade

```diff
@@ -329,7 +315,7 @@ figure
 
   **Two-way data binding** is an important fourth form
   that combines property and event binding in a single notation, using the `ngModel` directive.
-  We didn't have a two-way binding in the `HeroListComponent` template;
+  There isn't a two-way binding in the `HeroListComponent` template;
```

Comment:
Let's rephrase this to be clearer. (And actually, I don't see a need to explicitly say that HLC doesn't have two-way binding.) Maybe:

Here's an example of two-way binding from the `HeroDetailComponent` template: